activerecord/lib/active_record.rb

```diff
@@ -381,6 +381,50 @@ def self.global_executor_concurrency # :nodoc:
   singleton_class.attr_accessor :dump_schemas
   self.dump_schemas = :schema_search_path
 
+  singleton_class.attr_accessor :db_config_handlers # :nodoc:
```

Comment:
As it's global and not inheritable / overridable by model classes or anything, I'd be tempted to store this (and the register method) on `DatabaseConfigurations`, just to move some of the name into the namespace. Might also keep the [fairly special-case] docs together with their likely usage/context too?

Reply:
Good idea, I made this change.